src/IIIFPresentation/API/Infrastructure/PresentationController.cs

```diff
@@ -33,7 +33,7 @@ protected UrlRoots GetUrlRoots()
         return new UrlRoots
         {
             BaseUrl = Request.GetBaseUrl(),
-            ResourceRoot = Settings.ResourceRoot.ToString()
+            ResourceRoot = Settings.ResourceRoot!.ToString()
```

Comment:
This PR marks `ResourceRoot` as nullable but using `!` here - can you guarantee it has a value here? If so can `ResourceRoot` be marked as not-nullable on settings object?